### PR TITLE
Create onboarding package with client

### PR DESCRIPTION
**Pull request by matthewcarlreetz**

### CODE REVIEW

File: packages/onboarding/src/Client.ts [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+
+type Response<T>={
+  code: number
+  data: T | null
+  success: boolean
+  errorMessage?: string
+  errors?: Array<any>
+}
+
+export default class Client {
```
Comment (allenan) [inline packages/onboarding/src/Client.ts:13]: could having this top level class named `Client` get confusing with the `Client` exported out of `@helium/http`?

Comment (matthewcarlreetz) [reply packages/onboarding/src/Client.ts:13]: Likely. I'll rename.

File: packages/onboarding/src/Client.ts [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+  constructor(baseURL: string = DEWI_ONBOARDING_API_BASE_URL) {
+    this.axios = axios.create({
+      baseURL,
+    })
+  }
+
+  async execute<T>(method:Method, path: string, params: Object = {}) {
+    const query = qs.stringify(params)
+    let url = path
+    if (method === 'GET' && query.length > 0) {
```
Comment (allenan) [inline packages/onboarding/src/Client.ts:25]: could `GET` specific logic be handled in the `get` function? then there wouldn't be a need to add case logic here

Comment (matthewcarlreetz) [reply packages/onboarding/src/Client.ts:25]: yeah, I like that.

File: packages/onboarding/package.json [hunk 3]
```diff
@@ -0,0 +1,39 @@
+{
+  "name": "onboarding",
```
Comment (allenan) [inline packages/onboarding/package.json:2]: think this should be `@helium/onboarding`

**Review by allenan — APPROVED** [review]
looks good!

**Pull request merged**